### PR TITLE
Create serialization.md

### PR DESCRIPTION
**Pull request by sigilante**

### CODE REVIEW

File: content/language/hoon/reference/serialization.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+The main tools from `/sys/hoon` for this are:
+
+- [`++cue`](https://docs.urbit.org/language/hoon/reference/stdlib/2p#cue), unpack a jammed noun
+- [`++jam`](https://docs.urbit.org/language/hoon/reference/stdlib/2p#jam), pack a jammed noun
+- [`++mat`](https://docs.urbit.org/language/hoon/reference/stdlib/2p#mat), length-encode a noun
+- [`++rub`](https://docs.urbit.org/language/hoon/reference/stdlib/2p#rub), length-decode a noun
+
+`++jam` and `++cue` are critically important for noun communication operations, including the `%lick`
+vane and
```
Comment (mopfel-winrux) [inline content/language/hoon/reference/serialization.md:16]: %lick and %khan vane

File: content/language/hoon/reference/serialization.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+(`++cue` distinguishes the bit length from the value by unary until the first `0`.)
+
+```hoon
+> `@ub`(jam ~)
+0b10
+::  start at LSB, so `0` for atom, `1` for length, `0` for value (head-trimmed zero)
+
+> `@ub`(jam 1)
+0b1100
+::  start at LSB, so `0` for atom, `01`
```
Comment (mopfel-winrux) [inline content/language/hoon/reference/serialization.md:33]: `1` for length, `1` for value

**Comment by mopfel-winrux** [comment]
A description of how `+jam` and `+cue` work would be good.

**Comment by sigilante** [comment]
Added `+jam`, `+cue`, `+rub` descriptions with close commentary.

With an approving review, this is ready to merge.

**Comment by tamlut-modnys** [comment]
This line isn't rendering properly in the vercel deployment:

> ++jam and ++cue are critically important for noun communication operations, including the %lick vane and

**Comment by sigilante** [comment]
I think it was stale:  https://docs-urbit-isb327rol-urbit.vercel.app/language/hoon/reference/serialization

**Comment by tamlut-modnys** [comment]
Is there supposed to be documentation for ++mat?

There's ++cue ++jam ++mat ++rub at the top but only documentation for ++cue ++rub ++jam

Otherwise looks good to me.

**Review by tamlut-modnys — COMMENTED** [review]
Could use documentation for ++mat

**Pull request merged**